src/components/Tag.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './tag.sass'
+
+interface TagProps {
+	value?: string
+	size?: string
+}
+
+export default class Tag extends React.PureComponent<TagProps> {
+	render() {
+		if (!this.props.value) {
+			return null
+		}
+		return (
+			<div className={this.props.size ? 'tag tag-' + this.props.size : 'tag tag-medium'}>
+				{this.props.value}
+			</div>
+		)
+	}
+}
```

Comment:
It is not globally yet. But in React I name classes same as component name - with capital first letter.

- `tag.sass` => `Tag.sass`
- `.tag` => `.Tag`
- dont prefix modifier classes with `tag`, better like `view-size-{size}.`
- if the `Tag` would have for example delete button, I would name its class `Tag-deleteButton`, which can also get modifier classes. The delete button could have its icon which i would name `Tag-deleteButton-icon`.

```html
<div className="Tag view-size-medium">
  <button className="Tag-deleteButton view-disabled">
    <img className="Tag-deleteButton-icon" />
  </button>
</div>
```